packages/rocketchat-lib/server/methods/unblockUser.js

```diff
@@ -0,0 +1,22 @@
+Meteor.methods({
+	unblockUser({rid, blocked}) {
+
+		check(rid, String);
+		check(blocked, String);
+
+		if (!Meteor.userId()) {
+			throw new Meteor.Error('error-invalid-user', 'Invalid user', { method: 'blockUser' });
+		}
+
+		const subscription = RocketChat.models.Subscriptions.findOneByRoomIdAndUserId(rid, Meteor.userId());
+		const subscription2 = RocketChat.models.Subscriptions.findOneByRoomIdAndUserId(rid, blocked);
+
+		if (!subscription || !subscription2) {
+			throw new Meteor.Error('error-invalid-room', 'Invalid room', { method: 'blockUser' });
+		}
+
+		RocketChat.models.Subscriptions.unsetBlockedByRoomId(rid, blocked, Meteor.userId());
```

Comment:
but it would be harder to display on channels lists, and we could use this to ban people on public channels with little changes.